opba-api-security/src/main/java/de/adorsys/opba/api/security/internal/service/TppAuthResponseCookieTemplate.java

```diff
@@ -17,4 +17,13 @@
                 .path(cookieProperties.getPath())
                 .maxAge(cookieProperties.getMaxAge());
     }
+
+    public ResponseCookie.ResponseCookieBuilder redirectToAspspBuilder(String name, String value) {
```

Comment:
Same here duplicates above function, pass the age as an argument to the common function to redirectToAspspBuilder and builder. Also `cookieProperties.getPath()` is wrong as this cookie should reside only on `fromAspsp endpoint set` and not entire API

Reply:
done